.changeset/odd-apes-guess.md

```diff
@@ -0,0 +1,10 @@
+---
+"@primer/react": major
+---
+
+`PageLayout` has been moved from the `drafts` bundle to the main bundle. To upgrade, rewrite your imports accordingly:
```

Comment:
Minor formatting suggestion:

```suggestion
`PageLayout` is being graduated from the `drafts` bundle to the `main` bundle. 

To upgrade, rewrite your imports accordingly:
```

I think we should opt to use graduated as it asserts the intention behind the move.